clients/lsp-kotlin.el

```diff
@@ -260,6 +271,10 @@ to Kotlin."
   :initialized-fn (lambda (workspace)
                     (with-lsp-workspace workspace
                       (lsp--set-configuration (lsp-configuration-section "kotlin"))))
+  :initialization-options (lambda ()
+                            (if lsp-kotlin-ondisk-cache-enabled
+                                (list :storagePath lsp-kotlin-ondisk-cache-path)
+                              (list)))
```

Comment:
I think this can just be `nil`? Or:

```elisp
(when lsp-kotlin-ondisk-cache-enabled
  (list :storagePath lsp-kotlin-ondisk-cache-path))
```

Reply:
You are right, off course 🙂 For some reason I was convinced that lsp-mode didn't like nil in places like these. Unsure where I got that from 😆